examples/gateway/kuard/kuard.yaml

```diff
@@ -37,20 +37,21 @@ spec:
   type: ClusterIP
 ---
 kind: HTTPRoute
-apiVersion: networking.x-k8s.io/v1alpha1
+apiVersion: gateway.networking.k8s.io/v1alpha2
```

Comment:
I have this change in https://github.com/projectcontour/contour/pull/4122 but had been holding off until closer to release so as to not break the existing guide against our latest released version. Maybe create a `-va1lpha2` version for now?